node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageReceiver.scala

```diff
@@ -294,7 +303,8 @@ class PeerMessageReceiver(
         logger.debug(s"Handling expected response for ${msg.commandName}")
         val expectedResponseCancellable =
           system.scheduler.scheduleOnce(nodeAppConfig.queryWaitTime)(
-            Await.result(onResponseTimeout(msg), 10.seconds))
+            Await.result(node.peerManager.sendResponseTimeout(peer, msg),
```

Comment:
Needed as this future needs to be called with the then (after 10s) state. Not sure if there's a better way.